provisioning_templates/snippet/ansible_tower_callback_script.erb

```diff
@@ -0,0 +1,9 @@
+<%#
+kind: snippet
+name: ansible_tower_callback_script
+-%>
+#!/bin/sh
+
+echo "Calling Ansible AWX/Tower provisioning callback..."
+/usr/bin/curl -v -k -s --data "host_config_key=<%= host_param('ansible_host_config_key') %>" https://<%= host_param('ansible_tower_fqdn') %>/api/v2/job_templates/<%= host_param('ansible_job_template_id') %>/callback/
```

Comment:
Maybe change to host param `ansible_tower_fqdn` to `ansible_tower_url` to allow for `http` and `https` as schemas?

Reply:
I'd honestly not want this over plain text. HTTPS is getting so ubiquitous and that also means the user has to specify the full path. Note that the certificates aren't checked so you can even host it on a self signed cert.